modules/ossm-federation-config-export.adoc

```diff
@@ -0,0 +1,152 @@
+  - type: LabelSelector
+    labelSelector:
+      namespace: red-mesh-bookinfo
+      Selector:
+        matchLabels:
+          export-service: “true”
+      alias: # exported as if they were in the bookinfo namespace
+        namespace: bookinfo
+----
+
+.ExportServiceSet parameters
+[options="header"]
+[cols="l, a, a"]
+|===
+|Parameter |Description |Values
+|metadata:
+  name:
+|Name of the ServiceMeshPeer you are exposing this service to.
+|Must match the `name` value for the mesh in the `ServiceMeshPeer` resource.
+
+|metadata:
+  namespace:
+|Name of the project/namespace you are exporting this service from, that is, a namespace that is listed on this service mesh's member roll (SMMR) .
+|
+
+|spec:
+  exportRules:
+  -type:
+|Type of rule that will govern the export for this service. The first matching rule found for the service will be used for the export.
+|`NameSelector`, `LabelSelector`
+
+|spec:
+  exportRules:
+  -type: nameSelector
+    NameSelector:
+      namespace:
+      name:
+|To create a `NameSelector` rule, specify the `namespace` of the service and the `name` of the service as defined in the `Deployment` resource.
+|
+
+|spec:
+  exportRules:
+  -type: NameSelector
+    NameSelector:
+      alias:
+        namespace:
+        name:
+|To create a `NameSelector` rule that uses an alias for the service, after specifying the `namespace` and `name` for the service, then specify the alias for the `namespace` and the alias to be used for `name` of the service.
+|
+
+|spec:
+  exportRules:
+  -type: LabelSelector
+    LabelSelector:
+      namespace: <exportingMesh>
+      Selector:
+        matchLabels:
+        <label>: "true"
+|To create a `LabelSelector` rule, specify the `namespace` of the service and specify the `label` defined in the `Deployment` resource. In the example above, the label is `export-service`.
+|
+
+|spec:
+  exportRules:
+  -type: LabelSelector
+    LabelSelector:
+      namespace: <exportingMesh>
+      Selector:
+        matchLabels:
+          <label>: "true"
+      alias:
+        namespace:
+        name:
+|To create a `LabelSelector` rule that uses an alias for the service, after specifying the `namespace` and `label`, then specify the alias to be used for `name` or `namespace` of the service. In the example above, the alias is `bookinfo`.
+|
+|===
+
+//PLEASE CHECK THESE EXAMPLES
+
+.Export services with the name "ratings" from all namespaces in the red-mesh to blue-mesh.
+[source,yaml]
+----
+kind: ExportedServiceSet
+apiVersion: federation.maistra.io/v1
+metadata:
+  name: blue-mesh
+  namespace: red-mesh-system
+spec:
+  exportRules:
+  - type: NameSelector
+    nameSelector:
+      namespace: *
+      name: ratings
+----
+
+.Export all services from the west-data-center namespace to both green-mesh and blue-mesh
+[source,yaml]
+----
+kind: ExportedServiceSet
+apiVersion: federation.maistra.io/v1
+metadata:
+  name: green-mesh, blue-mesh
```

Comment:
```suggestion
  name: green-mesh
```
`name` can only have a single value.  `ExportedServiceSet` must be defined for each peer.